frontend/src/Pages/Home/InputForm/InputForm.tsx

```diff
@@ -99,6 +109,43 @@ const InputForm = () => {
     fetchData();
   }, []);
 
+  const fetchTags = async () => {
+    await axios({
+      method: "GET",
+      url: `/api/tag`,
+      baseURL: baseBackendUrl,
+    })
+      .then((response) => {
+        const res = response.data.map((tag: { name: string }) => tag.name);
+        setSkills(res);
+      })
+      .catch((error) => {
+        if (error.response) {
+          console.log(error.response);
+          console.log(error.response.status);
+          console.log(error.response.headers);
+        }
+      });
+  };
+
+  useEffect(() => {
```

Comment:
ideally we should combine the two useEffect functions because they both operate when the page is loaded and don't have any dependencies.

Reply:
combined now